codegen/src/main/java/software/amazon/awssdk/codegen/poet/builder/BaseClientBuilderClass.java

```diff
@@ -479,6 +480,13 @@ private MethodSpec finalizeServiceConfigurationMethod() {
                .addCode("    .fipsEnabled(c.get($T.FIPS_ENDPOINT_ENABLED))", AwsClientOption.class)
                .addCode("    .build());");
 
+        if (model.getMetadata().isJsonProtocol()) {
+            if (model.getCustomizationConfig().getEnableFastUnmarshaller()) {
+                builder.addStatement("builder.option($1T.ENABLE_FAST_UNMARSHALLER, true)",
```

Comment:
This should be false right?

Reply:
I don't think so, why do you think it should be?

Reply:
Nevermind, I got confused!